client/src/shared/components/Sidebar.jsx

```diff
@@ -13,6 +13,15 @@ function Sidebar() {
 
   return (
     <div className={`${styles.sidebar} flex flex-col`}>
+      <ProtectedElement minLevel={2}>
```

Comment:
This should not be here? Someone with no permissions should be able to view it. This causes it to be level 2+ only

Reply:
fixed